### ISSUE TITLE
Cant see other rp profiles

### ISSUE THREAD

**Issue by Ellypse** [Ellypse/Total-RP-3#129]
Submitter: John
When running TRP3 (V 1.2.9) I am having an issue where Only i can access my own profile. All of the buttons and menu options for the addon are there but I am unable to see other's profiles and others cannot see mine. Running different addons has done nothing as I have only enabled trp3 and it is not working.

**Comment by PyroBird** [comment]
I am having the same problem but I think I managed to find the problem. No clue how to fix it though.
![world of warcraft screenshot 2018 06 02 - 22 04 40 97](https://user-images.githubusercontent.com/39870474/40883043-3fbaf4f0-66b1-11e8-8196-d2e9ce2836a8.png)  
Image may be hard to read but in the menu, under settings and modules setting three addons in particular are not running. (Check to see if this is the same for you) WIM, Tinytooltip and Prat.  I have no idea what to do as I have uninstalled it and reinstalled it three times.
